.github/workflows/rit.yml

```diff
@@ -0,0 +1,46 @@
+name: Rootstock Integration Tests
+
+on:
```

Comment:
question:

I think the issue with the pull_request trigger was the source branch that was set to all with **. Don't we want to run it at least once for a PR opened inside our organization? Maybe add an if on the job rootstock-integration-tests with something like:

    if: github.event.pull_request.head.repo.full_name == 'rsksmart/rskj'

Buuut, I don't see it as a big deal, because we can always run manually and we were already doing that triggering with a comment. So, to me, the way that it's set now it's very good and more than enough. 😃

Reply:
nice suggestion! adding a condition for PRs from `rsksmart/rskj` is a great idea for the future. For now, I think we should stick with manual and push event triggers to keep things simple and efficient. We can revisit this if we see a need for more automated PR testing